src/libraries/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxyGenerator.cs

```diff
@@ -57,11 +57,18 @@ internal static class DispatchProxyGenerator
         // This approach is used to prevent regenerating identical proxy types for identical T/Proxy pairs,
         // which would ultimately be a more expensive leak.
         // Proxy instances are not cached.  Their lifetime is entirely owned by the caller of DispatchProxy.Create.
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+            Justification = "Use of the MakeGenericMethod is done in a trim-safe way in DispatchProxyGenerator.")]
+        [SuppressMessage("Performance", "CA1810", Justification = "Requires to suppress a trim warning")]
```

Comment:
Can we instead put the `UnconditionalSuppressMessage` targeting the cctor? That way we don't have to add an explicit static ctor.